spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>spring-cloud-gcp-starters</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-gcp-starter-sql</artifactId>
+
+	<dependencies>
+		<dependency>
```

Comment:
That's different story.
We talk here about `application.properties/yml` definition.
The `@Value` is a user of properties already.
And that is already up to IntelliJ question... Far away outside of scope of this PR.

Only what I want here is to have that dependency.
Thanks